### PR TITLE
Add extrinsic to update trading pair config 

### PR DESCRIPTION
**Pull request by felixfaisal**
This PR will add an extrinsic to update pair config and it contains the unit tests for the extrinsic 

Please Note: 
Currently, In this PR we are using primitives of branch `polkadot-v0.9.28-ingress`. **Do not merge** until the primitives are update with changes and the branch here is updated to `polkadot-v0.9.28` 

**Update**: Using the branch `polkadot-v0.9.28` can be merged if approved

### CODE REVIEW

File: pallets/ocex/src/lib.rs [hunk 1]
```diff
@@ -391,6 +393,55 @@ pub mod pallet {
...
+			T::GovernanceOrigin::ensure_origin(origin)?;
+			ensure!(base != quote, Error::<T>::BothAssetsCannotBeSame);
+			ensure!(
+				<TradingPairs<T>>::contains_key(base, quote),
+				Error::<T>::TradingPairNotRegistered
+			);
+			let trading_pair_info = TradingPairConfig {
+				base_asset: base,
+				quote_asset: quote,
+				min_price: Decimal::from(min_order_price.saturated_into::<u128>())
```
Comment (Gauthamastro) [inline pallets/ocex/src/lib.rs:504]: This will overflow.

**Review by Gauthamastro — DISMISSED** [review]

File: pallets/ocex/src/lib.rs (hunk 1 above)
Comment (felixfaisal) [reply pallets/ocex/src/lib.rs:504]: Now ensuring that these params do not exceed `10^27`

Comment (Gauthamastro) [reply pallets/ocex/src/lib.rs:504]: You also need to check for these if values are not zero

**Review by Gauthamastro — CHANGES_REQUESTED** [review]
Checks for non-zero and is_positive() are needed

File: pallets/ocex/src/lib.rs [hunk 2]
```diff
@@ -196,6 +196,8 @@ pub mod pallet {
 		OnchainEventsBoundedVecOverflow,
 		/// Overflow of Deposit amount
 		DepositOverflow,
+		/// Trading Pair is not registed for updating
+		TradingPairNotRegistered,
```
Comment (serhii-temchenko) [inline pallets/ocex/src/lib.rs:198]: Do we need 2 different error types for the same error? we have already `StorageOverflow` which, if I'm not mistaken, doing the same thing. Mby we could use one common error? If it's needed we can rename `StorageOverflow` to more common name e.g. `AmmountOwerflow` and use it in two places?

File: pallets/ocex/src/tests.rs [hunk 3]
```diff
@@ -322,6 +322,102 @@ fn test_register_trading_pair_trading_pair_already_registered() {
...
+				1_u128.into(),
+				100_u128.into(),
+				100_u128.into(),
+				10_u128.into()
+			),
+			BadOrigin
+		);
+	});
+}
+
```
Comment (serhii-temchenko) [inline pallets/ocex/src/tests.rs:653]: I think should be added tests to test added in this PR assertions (`ensures` e.g. [this](https://github.com/Polkadex-Substrate/Polkadex/blob/5cf79b53346c41deb061c7c27a4f8e28fc6d9da0/pallets/ocex/src/lib.rs#L441-L444)). Bassically all places reported by `codecov`.

**Review by serhii-temchenko — CHANGES_REQUESTED** [review]

File: pallets/ocex/src/lib.rs (hunk 2 above)
Comment (felixfaisal) [reply pallets/ocex/src/lib.rs:198]: Updated ✅

File: pallets/ocex/src/tests.rs (hunk 3 above)
Comment (felixfaisal) [reply pallets/ocex/src/tests.rs:653]: Updated ✅

File: pallets/ocex/src/lib.rs (hunk 1 above)
Comment (felixfaisal) [reply pallets/ocex/src/lib.rs:504]: Updated ✅

**Review by serhii-temchenko — DISMISSED** [review]
Please resolve conflicts.

**Review by serhii-temchenko — DISMISSED** [review]

**Review by Gauthamastro — APPROVED** [review]

**Review by serhii-temchenko — APPROVED** [review]

**Pull request merged**